redis_enterprise/assets/dashboards/redis_enterprise_overview.json

```diff
@@ -6,7 +6,7 @@
       "id": 325090501093728,
       "definition": {
         "title": "About Redis Enterprise",
-        "banner_img": "https://raw.githubusercontent.com/DataDog/integrations-extras/58e0302ab52d7e1425cdcc5eb14dd16b9898db9e/redis_enterprise/assets/logos/logo.svg",
+        "banner_img": "https://github.com/DataDog/integrations-extras/blob/master/redis_enterprise/assets/logos/logo.svg",
```

Comment:
This image will not appear. This will need to be the raw image like the original link you provided. To get this, you can go to the image in your integration folder in the ints-extras repo, open image in a new tab, and copy the link. Ensure it has: raw.githubusercontent in it